src/modules/projectlist.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable no-undef */
+/*  eslint-disable  no-unused-vars */
```

Comment:
you shouldn't be having unused variables if projectList and globalTaskId aren't used here, then this means you can optimize the code better and take them to the right location